src/components/TipCalculator.vue

```diff
@@ -0,0 +1,88 @@
+<template lang="pug">
+  .container
+    .row
+      .col-12
+        h2 Tip Calculator
+
+    b-form
+      .row
+        .col
+          b-form-group(
+            id="input-billAmount"
+            label="Bill Amount:"
+            label-for="billAmount"
+            description="Enter your bill amount")
+            b-form-input(
+              id="billAmount"
+              v-model="form.billAmount"
+              type="number"
+              required
+              placeholder="0.00")
```

Comment:
Even though I've never seen Vue before, your code was really easy to follow. This is realllyyyy minor, but is there a reason you use `0.00` as a placeholder? I'm guessing it's because you want to ensure this is read as a float, but from a usability perspective, it would be better if the field was empty so we can just start typing rather than have to remove the 0.

Reply:
meant to remove, just missed it